src/aircrafts/F16/propulsion.jl

```diff
@@ -202,6 +202,7 @@ Reimplemented from:
 function calculate_engine(eng::F16Engine, fcs::FCS, aerostate::AeroState,
                           state::State; consume_fuel=false)
 
+    get_thrust(fcs) = get_value(fcs.cpl)        # temporary solution    # TODO
```

Comment:
what that this `# temporary solution   # TODO` mean?

Reply:
sorry i didnt write that clearly. i added this here temporarily because propulsion.jl is imported before  fcs.jl and this function is defined in fcs.jl.  this is temporary until this function is included in module/fcs.jl as you have mentioned in comment of that function.